source/common/world/Dimension.hpp

```diff
@@ -35,6 +35,8 @@
 
 #include <sol/sol.hpp>
 
+#include <DimensionPhysics.hpp>
```

Comment:
`#include "DimensionPhysics.hpp"` (always use quotes when the file is local to the project)